tox.ini

```diff
@@ -45,7 +45,7 @@ commands=
     native-state-metropolis: pytest {posargs:tests/json-fixtures/test_state.py -k Metropolis}
     lightchain_integration: pytest --integration {posargs:tests/trinity/integration/test_lightchain_integration.py}
 
-deps = .[p2p,trinity,eth-extra,test]
+deps = .[eth,p2p,trinity,eth-extra,test]
```

Comment:
This is confusing to me: `.` should already be installing those packages:
https://github.com/cburgdorf/py-evm/blob/db8bf2e1003bb18542a3e36ff46fbffabfddc808/setup.py#L102

Reply:
Thanks for pointing that out, now I'm confused as well. Let's see https://github.com/ethereum/py-evm/pull/1417